>From 826984dee547b41238dc9572a1c756506a3c9148 Mon Sep 17 00:00:00 2001
From: Jason Woodward <jason.woodward@timesys.com>
Date: Sat, 11 Jun 2011 19:09:53 -0400
Subject: [PATCH] Fix e500 fpu libm build

Define libm_ARCH_fpu_OUT in libm/powerpc/e500/Makefile.arch as it is used
in libm/powerpc/e500/fpu/Makefile.arch

 libm_ARCH_OBJ:=$(patsubst $(libm_ARCH_fpu_DIR)/%.c,$(libm_ARCH_fpu_OUT)/%.o,$(libm_ARCH_SRC))

This appears to have been broken since cd3a494e99fa4bcad1c2a621b71361005528bead

Signed-off-by: Jason Woodward <jason.woodward@timesys.com>
---
 libm/powerpc/e500/Makefile.arch |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/libm/powerpc/e500/Makefile.arch b/libm/powerpc/e500/Makefile.arch
index 912ce7f..febde67 100644
--- a/libm/powerpc/e500/Makefile.arch
+++ b/libm/powerpc/e500/Makefile.arch
@@ -6,5 +6,6 @@
 #
 
 libm_ARCH_fpu_DIR := $(libm_SUBARCH_DIR)/fpu
+libm_ARCH_fpu_OUT := $(libm_SUBARCH_OUT)/fpu
 -include $(libm_ARCH_fpu_DIR)/Makefile.arch
 
-- 
1.7.0.4

